3rdparty/pine/pine_server.h

```diff
@@ -4,6 +4,12 @@
 
 #pragma once
 
+// IPC uses a concept of "slot" to be able to communicate with multiple
+// emulators at the same time, each slot should be unique to each emulator to
+// allow PnP and configurable by the end user so that several runs don't
+// conflict with each others
+#define IPC_DEFAULT_SLOT 28012
+
```

Comment:
you could also remove the commented include line below. I don't know why it exists.